Example/ViewController.m

```diff
@@ -69,14 +56,89 @@ - (void)zoomOut
     [self.logVC decreaseFontSize];
 }
 
-- (void)runTapped
+- (IBAction)buttonClicked:(id)sender
+{
+    UIAlertController *actionSheet = [UIAlertController alertControllerWithTitle:@"digi.me" message:@"Choose Consent Access flow" preferredStyle:UIAlertControllerStyleActionSheet];
+    [actionSheet addAction:[UIAlertAction actionWithTitle:@"Cancel" style:UIAlertActionStyleCancel handler:^(UIAlertAction *action) {
+        [self dismissViewControllerAnimated:YES completion:nil];
+    }]];
+
+    [actionSheet addAction:[UIAlertAction actionWithTitle:@"Ongoing Consent Access" style:UIAlertActionStyleDefault handler:^(UIAlertAction *action) {
+        [self runOngoingAccessFlow];
+        [self dismissViewControllerAnimated:YES completion:nil];
+    }]];
+
+    [actionSheet addAction:[UIAlertAction actionWithTitle:@"Legacy Consent Access" style:UIAlertActionStyleDefault handler:^(UIAlertAction *action) {
+        [self runLegacyFlow];
+        [self dismissViewControllerAnimated:YES completion:nil];
+    }]];
+
+    [self.navigationController presentViewController:actionSheet animated:YES completion:nil];
+}
+
+-  (void)runOngoingAccessFlow
 {
+    if (self.accessToken && [[NSDate date] compare: self.accessToken.expiresOn] == NSOrderedAscending)
+    {
+        [self ongoingAccessRetrieveData];
+    }
+    else
+    {
+        [self beginOngoingAccess];
+    }
+}
+
+- (void)beginOngoingAccess
+{
+    self.configuration = [self createSampleConfiguration];
+    
     if (self.configuration)
     {
         self.dmeClient = nil;
         self.dmeClient = [[DMEPullClient alloc] initWithConfiguration:self.configuration];
     }
     
+    [self.logVC reset];
+    
+    DMEScope *scope = [self createSampleScopeForOneYearOfSocialData];
```

Comment:
It's great that we've been using this to test the implementation, but we don;t need it in the example app. Please take scoping out.